packages/terra-navigation-side-menu/src/orion-fusion-theme/NavigationSideMenuItem.module.scss

```diff
@@ -36,16 +36,19 @@
     --terra-navigation-side-menu-item-selected-before-background-image: none;
     --terra-navigation-side-menu-item-selected-background-size: auto;
     --terra-navigation-side-menu-item-selected-border-left: 0.5rem solid #007cc3;
-    --terra-navigation-side-menu-item-selected-chevron-color: #909496;
+    --terra-navigation-side-menu-item-selected-chevron-color: #1C1F21;
     --terra-navigation-side-menu-item-selected-color: #1c1f21;
     --terra-navigation-side-menu-item-selected-focus-background-color: #f4f4f4;
-    --terra-navigation-side-menu-item-selected-focus-chevron-color: #909496;
+    --terra-navigation-side-menu-item-selected-focus-chevron-color: #1C1F21;
```

Comment:
I think you forgot to revert this

Reply:
yeah. this variable is not used any more. Removed . Thanks
3017b29e248e24b6a7a1746aac6243fccca50af3